unitary/alpha/quantum_world.py

```diff
@@ -59,7 +59,7 @@ def clear(self):
         self.objects: List[QuantumObject] = []
```

Comment:
You can get rid of `self.objects` now since it's redundant information with used_object_keys.

Reply:
Done.  Changed objects to be a property.